rust/exchange_rate/canister_ids.json

```diff
@@ -0,0 +1,5 @@
+{
+  "exchange_rate": {
+    "ic": "464yd-nyaaa-aaaam-qag5a-cai"
+  }
+}
```

Comment:
The canister exchange_rate_assets is not mentioned here. Is this on purpose?

Reply:
This was because the frontend canister so far has only been tested locally, we haven't created it on mainnet (as the feature is not yet launched on mainnet).  Removed the canister_ids for mainnet for now.